src/components/ThoughtAnnotation.tsx

```diff
@@ -60,10 +60,10 @@ const mapStateToProps = (state: State, props: ThoughtAnnotationProps) => {
 
   const { cursor, cursorBeforeEdit, invalidState, editingValue, showHiddenThoughts } = state
 
-  const isEditing = equalPath(cursorBeforeEdit, props.thoughtsResolved)
+  const isEditing = equalPath(cursorBeforeEdit, props.path)
   const pathLive = isEditing
-    ? parentOf(props.path).concat(head(props.showContexts ? parentOf(cursor!) : cursor!)) as SimplePath
-    : props.path
+    ? parentOf(props.simplePath).concat(head(props.showContexts ? parentOf(cursor!) : cursor!)) as SimplePath
+    : props.simplePath
```

Comment:
Let's call this `simplePathLive` to avoid confusion. We should also return `simplePath: simplePathLive` instead of `path: pathLive`.